apps/src/gamelab/GameLab.js

```diff
@@ -939,7 +938,7 @@ GameLab.prototype.onDPadMouseMove = function (e) {
     clientY = e.touches[0].clientY;
   }
 
-  if (experiments.isEnabled('fourWayDPad')) {
+  if (this.mobileControlsConfig.dpadFourWay) {
```

Comment:
Is this PR also removing this experiment / enabling it for everyone?  Or was this leftover from a past PR?

Reply:
yes, it removes the experiment. @poorvasingal wants the 4-way to be the default now, and the new API allows you to change it to 8-way.